libs/coin-evm/src/signOperation.ts

```diff
@@ -60,7 +61,11 @@ export const buildSignOperation =
   }): Observable<SignOperationEvent> =>
     new Observable(o => {
       async function main(): Promise<void> {
-        const preparedTransaction = await prepareForSignOperation(account, transaction);
+        const preparedTransaction = await prepareForSignOperation({
+          account,
+          transaction,
+          isValidNonce: transaction.nonce !== DEFAULT_NONCE,
```

Comment:
I believe you meant without the `isValidNonce` param of the function, right?... That is what I had in mind and what is cleaner and simpler code-wise, the only reason not to, would be if there's any use case we can think of where comparing to the value of `DEFAULT_NONCE` is _not_ the right logic.
But otherwise, I would put the comparison inside the function and remove the param indeed. Anyway not super important :)

Reply:
Updated here 847a3a18f80781090fc485ecf3b18a9fce1479c0